collections/_posts/2022-11-23-FedAMP_Personalization_in_cross_silo_Federated_learning.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: review
+title: "FedAMP: Personalized Cross-Silo Federated Learning on Non-IID Data"
+tags:  federated learning, personalization, cross-silo
+author: "Manthe Matthis"
+cite:
+    authors: "Yutao Huang, Lingyang Chu, Zirui Zhou, Lanjun Wang,
+Jiangchuan Liu, Jian Pei, Yong Zhang"
+    title:   "Personalized Cross-Silo Federated Learning on Non-IID Data"
+    venue:   "AAAI 2021"
+pdf: "https://arxiv.org/pdf/2007.03797.pdf"
+---
+
+# Highlights
+
+* The authors propose a new federated personalization method,
+* The goal is to facilitate pairwise collaborations between institutions with similar data,
+* The method is based on an alternation between optimization of local losses and of a message passing regularizer,
+* The method can be applied to any supervised learning task (classification, segmentation, ...).
+
+# Method
+
+## Problem definition
+Consider a federated learning problem with $$M$$ institutions, each with $$n_m$$ samples.
+* Let's note the overall personalized objective function, with $$W:= [w_i]_{i<M}$$ the matrix of personalized model's parameters $$w_i\in\mathbb{R}^d$$
+
+$$ \mathcal{F}(W) := \sum_{i=1}^MF_i(w_i)$$
+
+* The message passing regularizer 
+
+$$ \mathcal{A}(W):= \sum_{i<j}^MA(||w_i-w_j||^2)$$
+
+The function $$A:[0, \infty) \rightarrow \mathbb{R}$$ is increasing, concave, $$A(0)=0$$, continuously differentiable and $$lim_{t\rightarrow 0^+}A'(t)$$ is finite (they use $$A(||w_i-w_j||^2) = 1 - e^{-||w_i-w_j||^2/\sigma}$$). **It measures
+the difference between $$w_i$$ and $$w_j$$ in a non-linear manner.**
```

Comment:
I would include _A(0)=0_ closer to the definition of the function itself, because on a first read I didn't understand it was part of an enumeration of properties of the function. So something like:
```suggestion
The function $$A:[0, \infty) \rightarrow \mathbb{R}$$, with $$A(0)=0$$, is increasing, concave, continuously differentiable and $$lim_{t\rightarrow 0^+}A'(t)$$ is finite (they use $$A(||w_i-w_j||^2) = 1 - e^{-||w_i-w_j||^2/\sigma}$$). **It measures
the difference between $$w_i$$ and $$w_j$$ in a non-linear manner.**
```